src/utils/getCirclePoints.ts

```diff
@@ -0,0 +1,20 @@
+import { circle } from 'bresenham-zingl';
+
+import { Point } from '../geometry';
+
+/**
+ * Get the coordinates of the points on a circle. The reference is the center of the circle.
+ * Caution: the points are not sorted.
+ *
+ * @param radius - Radius of the circle.
+ * @returns The coordinates of the points on a circle of given diameter.
+ */
+export function getCirclePoints(radius: number): Point[] {
```

Comment:
You created this utility but it isn't used anywhere in the code.

Reply:
I would actually need it, but I didn't know how to implement it properly. I need the points on a circle ordered, but my current implementation returns the points mixed up. Do you have an advice on how to get the coordinates of pixels on a circle with given radius?